### PR TITLE
Mozilla CA Certificates

### PR DESCRIPTION
**Pull request by aviks**
This packages up the .pem form of the Mozilla CA Certificates, downloaded via the LibCURL website.

### CODE REVIEW

File: M/MozillaCACerts/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "MozillaCACerts"
+version = v"1.0.0"
```
Comment (giordano) [inline M/MozillaCACerts/build_tarballs.jl:6]: What about using the date as version?

Comment (aviks) [reply M/MozillaCACerts/build_tarballs.jl:6]: I was thinking about that -- `2020.01.01`? Or will it be `2020.1.1`? Will these be sorted correctly?

Comment (giordano) [reply M/MozillaCACerts/build_tarballs.jl:6]: It'll be a `VersionNumber`, so it should be sorted as expected:
```julia
julia> v"2020.02.01"
v"2020.2.1"

julia> v"2020.02.01" < v"2020.11.01"
true
```

**Comment by staticfloat** [comment]
Congrats on getting pull #1000!

Any reason you want this as a JLL and not as a "normal" Julia package?

**Comment by giordano** [comment]
> Any reason you want this as a JLL and not as a "normal" Julia package?

Can `Artifacts.toml` deal with uncompressed files?  My understanding was that they kind of want a tarball, but I never tried in practice.

**Comment by staticfloat** [comment]
I meant just a package with the `.pem` file embedded directly within it.  :P

**Comment by giordano** [comment]
> I meant just a package with the `.pem` file embedded directly within it. :P

I can't believe you're suggesting to do this :scream:

**Comment by aviks** [comment]
> I meant just a package with the .pem file embedded directly within it. 

I mean, that is exactly what MozillaCACert_jll is. But using the BB tooling makes it much easier register and maintain this kind of package. (I hate the passive aggressive phrase "as you know", but probably never truer than for the previous sentence 😄) 

When I have to update this, its a five minute change to edit the `build_tarballs` here, vs download, check sha, git commit, push, verify, tag etc.

**Comment by staticfloat** [comment]
Yeah, I don't have strong feelings about it.  In my mind, the reason for JLL existence is the ability to deal with dependency chains nicely, set environment variables for binaries automatically, and to deal with artifacts across-platforms.  If it's just a small amount of textual data, I figure, why not just embed it into a package directly?  But if you feel that Yggdrasil is easier to work with than a normal Julia package, that's pretty high praise! :D

**Pull request merged**